path = /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/test/temp/129
implementation directory: /QUIC-FormalVerification/quic-implementations/quic-go/server/
implementation command: ./server -c /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/cert.pem -k /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/priv.key -p 4443 127.0.0.1
quic_server_test_0rtt
[<utils.IvyTest.IvyTest object at 0x7efc09af9ac8>]
./server -c /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/cert.pem -k /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/priv.key -p 4443 127.0.0.1
/QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_server_test_0rtt (29) ...
/QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/test/temp/129/quic_server_test_0rtt29.out
/QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/test/temp/129/quic_server_test_0rtt29.err
/QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/test/temp/129/quic_server_test_0rtt29.iev
implementation command: exec ./server -c /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/cert.pem -k /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/priv.key -p 4443 127.0.0.1
quic_process pid: 4572
iclient = 0
quic_server_test_0rtt
timeout 30 /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_server_test_0rtt_stream_app_close seed=883 the_cid=30 server_port=4443 server_cid=29 client_port=5045 client_port_alt=5046;sleep 1;timeout 30 /QUIC-FormalVerification/QUIC-Ivy/doc/examples/quic/build/quic_server_test_0rtt seed=883 the_cid=32 server_port=4443 server_cid=31 client_port=5045 client_port_alt=5046
client return code: 134
quic_process.terminate()
-15
pid is unassigned
FAIL
False
     Run  ... version_not_found
0      0  ...               0.0
1      1  ...               0.0
2      2  ...               0.0
3      3  ...               0.0
4      4  ...               0.0
..   ...  ...               ...
124  124  ...               0.0
125  125  ...               0.0
126  126  ...               0.0
127  127  ...               0.0
128  128  ...               0.0

[129 rows x 28 columns]
error: 1 tests(s) failed
